src/components/posts/PostsList.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+
+    <v-list
+        class="py-0"
+        outlined
+        rounded
+        width="100%"
+        color="transparent"
+    >
+      <template v-for="(post, index) in posts">
+        <v-list-item
+            :key="index"
+        >
+          <v-list-item-content>
+            <v-card
+                class="pa-5"
+                min-height="130"
+            >
+              {{ post.content }}
+            </v-card>
+          </v-list-item-content>
+          <v-btn
+              class="ma-2"
+              text
+              icon
+              color="blue lighten-2"
+              @click="giveReaction(post.id, 'like')"
+          >
+            <v-icon>mdi-thumb-up</v-icon>
+          </v-btn>
+          <v-btn
+              class="ma-2"
+              color="red lighten-2"
+              icon
+              text
+              @click="giveReaction(post.id, 'dislike')"
+          >
+            <v-icon>mdi-thumb-down</v-icon>
+          </v-btn>
+        </v-list-item>
+      </template>
+    </v-list>
+<v-bottom-navigation
+  fixed
+background-color="transparent">
+    <v-pagination
+        v-model="page"
+        :length="totalPages"
+        total-visible="3"
+        next-icon="mdi-menu-right"
+        prev-icon="mdi-menu-left"
+        @input="handlePageChange"
+    ></v-pagination>
+    </v-bottom-navigation>
+  </div>
+</template>
+
+<script>
+import $axios from '@/axios';
+
+export default {
+  name: 'PostList',
+  data() {
+    return {
+      posts: [],
+      page: 1,
+      totalPages: 0,
+      pageSize: 4,
```

Comment:
i would increase the page size to 8 even if that would cause some pages to be scrollable